### PR TITLE
change README to tell about coverage only, not full project config

### PR DESCRIPTION
**Pull request by geoffray-adde-sonarsource**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -34,14 +34,6 @@ You can take a look at the link:sonar-project.properties[sonar-project.propertie
 - https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
 - https://docs.sonarcloud.io/enriching/test-coverage/test-coverage-parameters/[Test coverage parameters]
 
-= MacOS\XCodeBuild
```
Comment (arseniy-sonar) [inline README.adoc:37]: Why do you propose to remove this part?

**Review by arseniy-sonar — CHANGES_REQUESTED** [review]
I don't see the point of removing the part about building the project

File: README.adoc (hunk 1 above)
Comment (arseniy-sonar) [reply README.adoc:37]: After f2f discussion. It is no longer relevant.

**Review by arseniy-sonar — APPROVED** [review]

File: README.adoc (hunk 1 above)
Comment (geoffray-adde-sonarsource) [reply README.adoc:37]: as agreed, closing this

**Pull request merged**